crates/swc_cli/src/commands/compile.rs

```diff
@@ -271,6 +253,36 @@ struct InputContext {
 
 #[swc_trace]
 impl CompileOptions {
+    fn build_transform_options(&self, file_path: &Option<&Path>) -> anyhow::Result<Options> {
+        let base_options = Options::default();
+        let base_config = Config::default();
+
+        let config_file = if let Some(config_file_path) = &self.config_file {
+            let config_file_contents = fs::read(config_file_path)?;
+            serde_json::from_slice(&config_file_contents)
+                .map_err(|e| anyhow::anyhow!("{}", e))
```

Comment:
Is this required?
I think `.context("")` is enough to convert an error to `anyhow::Error`

Reply:
yes, looks like old oversight.